docs/source/cli_help/traverse.rst

```diff
@@ -0,0 +1,81 @@
+.. role:: bash(code)
+
+.. _traverse:
+
+traverse
+--------
+Usage:
+::
+    git machete traverse [-F|--fetch] [-l|--list-commits] [-M|--merge]
```

Comment:
Hmm still dimmed for some reason

Reply:
I forgot to put here `code-block` like with the rest of the commands.